client/lib/importer/actions.js

```diff
@@ -9,7 +9,25 @@ const wpcom = require( 'lib/wp' ).undocumented();
 /**
  * Internal dependencies
  */
-import { actionTypes, appStates } from './constants';
+import {
+	IMPORTS_AUTHORS_SET_MAPPING,
+	IMPORTS_AUTHORS_START_MAPPING,
+	IMPORTS_FETCH,
+	IMPORTS_FETCH_FAILED,
+	IMPORTS_FETCH_COMPLETED,
+	IMPORTS_IMPORT_CANCEL,
+	IMPORTS_IMPORT_LOCK,
+	IMPORTS_IMPORT_RECEIVE,
+	IMPORTS_IMPORT_RESET,
+	IMPORTS_IMPORT_START,
+	IMPORTS_IMPORT_UNLOCK,
+	IMPORTS_START_IMPORTING,
```

Comment:
Maybe `IMPORTS_START` and `IMPORTS_IMPORT_START` vs `IMPORTS_IMPORT_START` and `IMPORTS_START_IMPORTING`